### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by liam-middlebrook**
There's probably a good amount more to add to the README before this is done. But I wanted to get this up so @decause is aware it's being worked on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # ofCourse - Open, Free Course(ware)
-
+ai
```
Comment (ryansb) [inline README.md]: ??

Comment (liam-middlebrook) [inline README.md]: :cat:

File: README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # ofCourse - Open, Free Course(ware)
-
```
Comment (ryansb) [inline README.md]: Some markdown parsers require a blank line after a # header, please put that back.

File: README.md [hunk 3]
```diff
@@ -28,7 +27,26 @@ Docs are available on [ReadTheDocs][rtfd]
...
+### Viewing the Site
+
+To view an ofCourse website you'll want to run the following commands.
+
+    $ virtualenv --no-site-packages -p python2 ofcourse_env
+    $ . ofcourse_env/bin/activate
+    $ cd <ofcourse_website_directory>
+    $ ofcourse run
+
+### Tests
```
Comment (ryansb) [inline README.md]: This should be an H2, not H3.

File: README.md [hunk 4]
```diff
@@ -28,7 +27,26 @@ Docs are available on [ReadTheDocs][rtfd]
...
+ofCourse website. To install virtualenv simply type the following
+command into a root shell.
+
+    $ pip install virtualenv
+
+### Viewing the Site
+
+To view an ofCourse website you'll want to run the following commands.
+
+    $ virtualenv --no-site-packages -p python2 ofcourse_env
```
Comment (ryansb) [inline README.md]: This won't always work because not everyone has a python2 binary. Probably should note that.

File: README.md [hunk 5]
```diff
@@ -28,7 +27,26 @@ Docs are available on [ReadTheDocs][rtfd]
...
+
+You'll probably want to have a python virtualenv setup for testing your
+ofCourse website. To install virtualenv simply type the following
+command into a root shell.
+
+    $ pip install virtualenv
+
+### Viewing the Site
+
+To view an ofCourse website you'll want to run the following commands.
```
Comment (ryansb) [inline README.md]: Maybe link to some explanation of venv instead of telling people to just run commands. Educational material, remember?

**Comment by ryansb** [comment]
@decause reminded me that these docs are already written http://ofcourse.readthedocs.org/en/latest/start.html and linked to literally 2 lines above your changes.

**Comment by liam-middlebrook** [comment]
Okay then, I'm going to close this issue for now.

**Pull request closed without merging**